iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -24,6 +24,7 @@
     public static final String SETTING_APNS_KEY = "apnsProdIntegrationName";
     public static final String SETTING_APNS_SANDBOX_KEY = "apnsSandboxIntegrationName";
     public static final String SETTING_LIST_ID = "listId";
+    public static final String SETTING_PARSE_USER_ATTRIBUTES = "parseUserAttributes";
```

Comment:
We may actually want to change this to something like "coerce strings to scalars" - I noticed that the same problem exists for attributes/`dataFields` of `CommerceItem`.